CS483/CS483/Kartaclysm/Data/Abilities/projectile_strike.xml

```diff
@@ -6,13 +6,23 @@
   <Transform>
     <Scale x="0.5" y="0.5" z="0.5" />
   </Transform>
+
+  <Children>
+    <GameObject>
+      <Transform>
+        <Scale x="0.5" y="0.5" z="0.5" />
+        <Translation x="0.0" y="0.05" z="0.0" />
+      </Transform>
+
+      <Components>
+        <GOC_3DModel>
+          <OBJFileName path="Assets/Abilities/bowling_ball.hobj"/>
+        </GOC_3DModel>
+      </Components>      
+    </GameObject>
+  </Children>
   
   <Components>
-
-    <GOC_3DModel>
-      <OBJFileName path="Assets/Abilities/bowling_ball.hobj"/>
-    </GOC_3DModel>
-
     <GOC_SphereCollider>
       <Offset x="0" y="0" z="0" />
       <Sphere radius="1.0" />
```

Comment:
If you shrink the size of the graphics, the collider radius should shrink too. You can set p_bDebugRender = true in the ComponentCollider.h constructor to see the size of the collider sphere